### PR TITLE
Implemented_the_intersect_function

### PR DESCRIPTION
**Pull request by obimmatimothy**
Look at the implementation of the intersect method for spheres

### CODE REVIEW

File: back_end/RayTracer/Models/Geometric/Sphere.cs [hunk 1]
```diff
@@ -23,7 +23,89 @@ public Sphere(Point3D center, float radius)
 
         public override HitInfo Intersect(Ray ray)
         {
-            throw new NotImplementedException();
+            HitInfo checker= new HitInfo();
+            double t;
+            //throw new NotImplementedException();
```
Comment (davidbenicek) [inline back_end/RayTracer/Models/Geometric/Sphere.cs]: Can we get rid off this commented out line of code?

**Review by davidbenicek — CHANGES_REQUESTED** [review]
Looks good apart from that one tiny line

**Comment by obimmatimothy** [comment]
I have Removed the Line of Commented Code

**Review by davidbenicek — APPROVED** [review]

**Review by othmankh — COMMENTED** [review]
The implementation seems to be right, but I wonder about the naming of variables, specially, the variable with name checker, are you sure about this naming ?

**Review by othmankh — APPROVED** [review]

**Comment by othmankh** [comment]
The merge is done.

**Pull request merged**